macros/get_campaign_history_columns.sql

```diff
@@ -2,21 +2,20 @@
 
 {% set columns = [
     {"name": "bidding_strategy", "datatype": dbt.type_string()},
-    {"name": "campaign_type", "datatype": dbt.type_string()},
     {"name": "creation_date", "datatype": dbt.type_timestamp()},
-    {"name": "daily_budget", "datatype": dbt.type_int()},
     {"name": "end_date", "datatype": "date"},
-    {"name": "id", "datatype": dbt.type_int()},
+    {"name": "id", "datatype": dbt.type_string()},
     {"name": "last_updated_date", "datatype": dbt.type_timestamp()},
     {"name": "name", "datatype": dbt.type_string()},
-    {"name": "placement", "datatype": dbt.type_string()},
     {"name": "portfolio_id", "datatype": dbt.type_int()},
-    {"name": "premium_bid_adjustment", "datatype": "boolean"},
     {"name": "profile_id", "datatype": dbt.type_int()},
     {"name": "serving_status", "datatype": dbt.type_string()},
     {"name": "start_date", "datatype": "date"},
     {"name": "state", "datatype": dbt.type_string()},
-    {"name": "targeting_type", "datatype": dbt.type_string()}
+    {"name": "targeting_type", "datatype": dbt.type_string()},
+    {"name": "budget", "datatype": dbt.type_string()},
+    {"name": "budget_type", "datatype": dbt.type_string()},
+    {"name": "effective_budget", "datatype": dbt.type_string()}
```

Comment:
It looks like in the materialized staging model the `budget` and `effective_budget` fields are actually a float datatype. What is the datatype of these fields in the source? We should match the macro datatype to how it is present in the source.

![image](https://github.com/fivetran/dbt_amazon_ads_source/assets/74217849/18d56e95-0437-4bed-86df-299365e79f1c)

Reply:
This was an oversight--in the source data it is also float. I have updated in the macro.